tasks/bbb-html5.yml

```diff
@@ -36,6 +36,15 @@
   notify:
     - restart bbb-web
 
+- name: increase max-old-space-size in frontends
+  become: true
+  replace:
+    path: /usr/share/meteor/bundle/systemd_start_frontend.sh
+    regexp: '(--max-old-space-size)=(\d*)'
+    replace: '\1={{ bbb_html5_frontend_max_old_space_size }}'
+  notify:
+    - restart bbb-html5
```

Comment:
maybe template this file?
or is that not necessary

Reply:
We have a similar task in tasks/config.yml. Maybe we should put them togehter, either in bbb-html5.yml or in config.yml